nixos/modules/services/misc/forgejo.nix

```diff
@@ -674,5 +680,5 @@ in
   };
 
   meta.doc = ./forgejo.md;
-  meta.maintainers = with lib.maintainers; [ bendlas emilylange ];
+  meta.maintainers = with lib.maintainers; [ bendlas emilylange pyrox0 ];
```

Comment:
Feel free to add yourself to `pkgs.forgejo` as well if you want (you asked in https://github.com/NixOS/nixpkgs/pull/306341#issuecomment-2073922593).

But please do so in another commit.
Atomic git commits and all.
You can use the same commit for `nixos/forgejo` and `forgejo` though, imho :)

Reply:
Removed from the first commit and made another commit adding myself to both.